github-release-install.sh

```diff
@@ -42,19 +42,15 @@ fi
 
 set -ouex pipefail
 
-API_JSON=$(mktemp /tmp/api-XXXXXXXX.json)
 API="https://api.github.com/repos/${ORG_PROJ}/releases/${RELTAG}"
 
 # retry up to 5 times with 5 second delays for any error included HTTP 404 etc
-curl --fail --retry 5 --retry-delay 5 --retry-all-errors -sL ${API} -o ${API_JSON}
-RPM_URLS=$(cat ${API_JSON} \
-  | jq \
-    -r \
-    --arg arch_filter "${ARCH_FILTER}" \
-    '.assets | sort_by(.created_at) | reverse | .[] | select(.name|test($arch_filter)) | select (.name|test("rpm$")) | .browser_download_url')
-for URL in ${RPM_URLS}; do
-  # WARNING: in case of multiple matches, this only installs the first matched release
-  echo "execute: rpm-ostree install \"${URL}\""
-  rpm-ostree install "${URL}"
-  break
-done
+curl --fail --retry 5 --retry-delay 5 --retry-all-errors -sL ${API} | \
```

Comment:
I don't see a compelling reason for the changes to this file.

In general, I prefer to not make style changes without a clear reason.

Please let me know if you have a strong reason for it, otherwise I request this file be reverted.

Reply:
My reason for this change was simplicity and tidiness. Eliminating the usage of a temporary file which is only cleaned up indirectly by another script seemed like an enhancement to me.

Reply:
> Eliminating the usage of a temporary file which is only cleaned up indirectly by another script, seemed like an enhancement to me.

Good call to clean up the temp file here rather than externally. I'd like to see that added at the end of this script rather than remove the use of the temp file. I intentionally kept the temp file rather than directly process the URL to aid in troubleshooting when problems occur.

The other changes are all fine, but it seems to be subjective improvements.

Reply:
Alright. This trap should do the trick.